mne/digitization/_utils.py

```diff
@@ -261,7 +261,7 @@ def _make_dig_points(nasion=None, lpa=None, rpa=None, hpi=None,
                         'kind': FIFF.FIFFV_POINT_EEG,
                         'coord_frame': FIFF.FIFFV_COORD_HEAD})
 
-    return _format_dig_points(dig)
+    return Digitization(_format_dig_points(dig))
```

Comment:
It looks like things would be a bit simpler if `_format_dig_points` returned a list cast to `Digitization` already. Then there wouldn't need to be a bunch of `Digitization(_format_dig_points(...))` calls

Reply:
agreed.

Reply:
Agreed, that was on my list already. But I can do it here. no prob.

Reply:
I would have liked better to do it the other way around. Prefer to import a constructor and have a call that does `something = MyConstructor()` than importing a private function and call `something = _my_obscure_func(foo)`.

But since I'll be doing subsequent PRs in these things, I'll change it back.